docs/modules/user-guide/partials/proc_migrating-projects.adoc

```diff
@@ -18,10 +19,12 @@ starterProjects:
   - name: "kafka-project"
     description: "Use this app to get a nodejs application for working with kafka"
   - name: "simple-project"
-    description: "Use this app to get a simple "hello world" nodejs application"
+    description: "Use this app to get a simple 'hello world' nodejs application"
 ----
 ====
 
 [role="_additional-resources"]
 .Additional resources
-* See link:https://github.com/che-incubator/devworkspace-api/issues/42[corresponding issue].
+For a deeper understanding of migrating projects, go to the GitHub issue on the following topic:
```

Comment:
I don't have that much experience with GitHub and describing it, but somehow the formulation "the GitHub issue on the following topic: XY" sounds weird (is there really only one GitHub issue on a certain topic?), also it is long. I'd suggest "Go to the following GitHub issue: XY"